.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
- explicit - Triggers Code Actions when explicitly saved. Same as true.

- https://code.visualstudio.com/updates/v1_85#_code-actions-on-save-and-auto